packages/app-mobile/components/screens/LogScreen.tsx

```diff
@@ -43,10 +57,36 @@ class LogScreenComponent extends BaseScreenComponent {
 		];
 	}
 
+	private refreshLogTimeout: any = null;
+	public override componentDidUpdate(_prevProps: Props, prevState: State) {
+		if ((prevState?.filter ?? '') !== (this.state.filter ?? '')) {
+			// We refresh the log only after a brief delay -- this prevents the log from updating
+			// with every keystroke in the filter input.
+			if (this.refreshLogTimeout) {
+				clearTimeout(this.refreshLogTimeout);
+			}
+			setTimeout(() => {
+				this.refreshLogTimeout = null;
+				void this.resfreshLogEntries();
+			}, 600);
+		}
+	}
+
+	public override componentDidMount() {
+		void this.resfreshLogEntries();
+
+		if (this.props.navigation.state.defaultFilter) {
+			this.setState({ filter: this.props.navigation.state.defaultFilter });
+		}
```

Comment:
This isn't used by this pull request &mdash; it's used by the mobile plugins pull request to show all logs that contain a specific plugin ID.